src/Enclave.edl

```diff
@@ -43,7 +52,10 @@ enclave {
          *      EINVAL - The value of an argument are invalid.
          */
 
-        public int occlum_ecall_init([in, string] const char* log_level, [in, string] const char* instance_dir, [in] const struct host_file_buffer* file_buffer);
+        public int occlum_ecall_init(
+            [in, string] const char* log_level, 
+            [in, string] const char* instance_dir, 
+            [in] struct host_file_buffer* file_buffer);
```

Comment:
[in] const struct host_file_buffer* file_buffer
add const keyword

Reply:
Done